test/hotspot/jtreg/vmTestbase/vm/compiler/complog/share/LogCompilationTest.java

```diff
@@ -29,7 +29,6 @@
 import nsk.share.log.LogSupport;
 import vm.share.options.Option;
 import vm.share.options.OptionSupport;
-import vm.share.process.ProcessExecutor;
```

Comment:
You got rid of this import, but ProcessExecutor is still referenced below. Is this file even referenced during test execution?

Reply:
The ProcessExecutor has been moved into this package, so it is local package now. 
Double checked that it is used and tests
 jtreg:open/test/hotspot/jtreg/vmTestbase/vm/compiler/complog
still pass.